source/connectors/kafka-sink-data-formats.txt

```diff
@@ -0,0 +1,346 @@
+       to manage schema definitions.
+
+   * - JSON with Schema
+     - JSON record structure with explicit schema information to
+       ensure the data matches the expected format.
+
+   * - JSON (plain)
+     - JSON record structure without an attached schema.
+
+   * - RAW JSON
+     - Serialized as a String. The JSON structure is not managed by
+       Kafka Connect.
+
+       .. note::
+
+          This format requires that the fields contain valid JSON even
+          when specifying the `StringConverter
+          <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/storage/StringConverter.html>`_
+
+.. note::
+
+   **RAW JSON** mode
+   requires the Strings to be valid JSON
+   Even when using RAW JSON mode, i.e. with `StringConverter
+   <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/storage/StringConverter.html>`_,
+   the expected Strings have to be valid JSON.
+
+For more information on Kafka data serialization, see `Kafka Connect
+Serialization Explained
+<https://www.confluent.io/blog/kafka-connect-deep-dive-converters-serialization-explained#choosing-serialization-format>`_.
+
+Configuration Example for AVRO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration provides example settings that use the
+**AVRO data format** with a Schema Registry:
+
+.. code-block:: properties
+
+   key.converter=io.confluent.connect.avro.AvroConverter
+   key.converter.schema.registry.url=http://localhost:8081
+
+   value.converter=io.confluent.connect.avro.AvroConverter
+   value.converter.schema.registry.url=http://localhost:8081
+
+For more information on using a Schema Registry, see `Schema Management
+<https://docs.confluent.io/current/schema-registry/index.html>`_.
+
+Configuration Example for JSON with Schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration provides example settings that use the
+**JSON with schema** data format. The Kafka topic data must be in
+JSON format and contain top-level objects ``schema`` and ``payload``.
+
+.. code-block:: properties
+
+   key.converter=org.apache.kafka.connect.json.JsonConverter
+   key.converter.schemas.enable=true
+
+   value.converter=org.apache.kafka.connect.json.JsonConverter
+   value.converter.schemas.enable=true
+
+Configuration Example for JSON without Schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration provides example settings that use the **JSON
+without schema** data format. The Kafka topic data must be in JSON format.
+
+.. code-block:: properties
+
+   key.converter=org.apache.kafka.connect.json.JsonConverter
+   key.converter.schemas.enable=false
+
+   value.converter=org.apache.kafka.connect.json.JsonConverter
+   value.converter.schemas.enable=false
+
+.. admonition:: Choose the appropriate format
+   :class: note
+
+   Top-level JSON objects named ``schema`` or ``payload`` are parsed
+   literally instead of as a validation schema.
```

Comment:
Yes, I think so. Is that the term commonly used for not treating it as schema metadata and instead creating objects?

Also, I think the admonition title isn't enough context so I updated the sentence to help clarify. Let me know what you think.

Reply:
The reason I mentioned explicitly is that using the word explicit is to contrast against the word 'implicit'.  For instance, the same (original) sentence could be roughly rephrased as "Top-level JSON objects that are named schema or payload are not parsed implicitly as a validation schema."

Vs if you had gone with the word "literally", it would be contrasting against the word figuratively. The following rendition of the above sentence doesn't make sense:
"Top-level JSON objects that are named schema or payload are not parsed figuratively as a validation schema."

That being said, I like the new phrasing a lot!

Reply:
I don't really know the correct term, but chose the term "literally" to describe a "string literal" rather than to use an expected format to parse out any special instructions (like a schema and payload). I agree that "explicitly" communicates this better though.